flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/hint/FlinkHintStrategies.java

```diff
@@ -122,8 +122,9 @@ public static HintStrategyTable createHintStrategyTable() {
                                 .build())
                 .hintStrategy(
                         StateTtlHint.STATE_TTL.getHintName(),
-                        // TODO support agg state ttl hint
-                        HintStrategy.builder(HintPredicates.JOIN)
+                        HintStrategy.builder(
```

Comment:
Nit: this is a comment on `STATE_TTL_NON_EMPTY_KV_OPTION_CHECKER`.
We'd better keep a unified error message template with other hints.

Suggested changes
```java
 litmus.check(
                        ttlHint.kvOptions.size() > 0,
                        "Invalid STATE_TTL hint, expecting at least one key-value options specified.");
```